### ISSUE TITLE
Grid Power issue when going to the Nether

### ISSUE THREAD

**Issue by DocAnders** [rwtema/extrautilities_old_issues#1765]
Hello,

I had an unusual problem, when I flew in the Nether my GP-network suddenly dropped to 6, I lost flight (squid ring) and went into lava on Hardcore. I had 9x solars and 4x water mills producting 16gp each (total of 73@day and 64@night), I had nothing else on my network, only thing draining power was the ring, The Resonater was in a chest, doesn't matter since it had no upgrades.

Regards Doc

**Comment by DocAnders** [comment]
Forgot to add, I am playing the Direwolf20 1.12 pack

**Comment by Jaysleazy** [comment]
I had the same issue, it turned out that my chunks keep unloading (unrelated to ExtraUtilities.)  may be worth checking out.

Scratch that, just went to nether with chunks loaded and it didn't work until I dropped it and picked it up again.

**Comment by DocAnders** [comment]
I found a solution, a temporary one at least, I made sure all of my gp generators are in the same chunk, popped down a chunkloader from IC2 (costs 1EU/t for each chunk, along with two solar panels and a batbox) I probably need to upgrade to an MFE, otherwise at some point during night I will plummet :)

**Comment by uecasm** [comment]
Yes, it seems like in the current version you have to both chunk-load your GP mills (which I didn't think was needed in previous MC versions, but maybe I'm misremembering) and also whenever you switch dimensions you have to mouse click and then put back the ring before you can use flight.

**Comment by beerrunn** [comment]
I was having this issue until i chunk loaded using the map. Shift click where you want to chunk load. I have an issue with the angel ring where after a few seconds up to like 60 seconds where the ring loses its power. I double tap the space bar and I regain flight. Very frustrating.
